frontend/src/components/ShootDetails/ShootInfrastructureCard.vue

```diff
@@ -25,9 +25,9 @@ limitations under the License.
           <v-icon color="cyan darken-2">cloud_queue</v-icon>
         </v-list-item-icon>
         <v-list-item-content>
-          <v-list-item-subtitle>Provider</v-list-item-subtitle>
+          <v-list-item-subtitle>{{vendorTitle}}</v-list-item-subtitle>
```

Comment:
```suggestion
          <v-list-item-subtitle>
            <vendor title extended :shootItem="shootItem">
          </v-list-item-subtitle>
          <v-list-item-title class="pt-1">
            <vendor extended :shootItem="shootItem">
          </v-list-item-title>
```

Reply:
✅ (did not accept suggestion here as the closing tag was missing, however I changed the code like you suggested)